src/data/markdown/translated-guides/en/01 Getting started/02 Installation/01 Troubleshooting.md

```diff
@@ -10,22 +10,49 @@ If you use such a distribution, you'll need to install them:
 
 ```bash
 sudo apt-get update && sudo apt-get install ca-certificates gnupg2 -y
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
-echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
 sudo apt-get update
 sudo apt-get install k6
 ```
 
+## apt-key is deprecated
+
+Some users might run into an `apt-key` deprecation warning while adding k6's repository key to their system's keyring: `Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`. Ubuntu and Debian deprecated the `apt-key` command indeed. In order to avoid it, and be future-proof, users should delete the existing `security@k6.io` existing repository key present on their system, and update their sources list accordingly.
```

Comment:
To avoid the repetition:

```suggestion
Some Ubuntu and Debian users might run into an `apt-key` deprecation warning while adding k6's repository key to their system's keyring: `Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`. In order to avoid it, and be future-proof, users should delete the existing `security@k6.io` repository key present on their system, and update their sources list accordingly.
```